src/lib/visualization/abstract_visualizer.hpp

```diff
@@ -180,6 +194,15 @@ class AbstractVisualizer {
     return wrapped_label.str();
   }
 
+  std::string _random_color() {
```

Comment:
What are the colors good for? When seeing the graph for the first time, I wondered if they had any semantics

Reply:
Na, they are there to allow you to associate edges and labels. Without labels it's hard to tell to which edge "l_suppkey = s_suppkey" belongs.